guides/common/modules/proc_configuring-direct-ad-integration-with-gss-proxy.adoc

```diff
@@ -70,7 +70,7 @@ euid = __ID_of_Apache_User__
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
 # systemctl restart gssproxy
-# systemctl enable --now gssproxy
+# systemctl enable gssproxy
```

Comment:
Isn't `# systemctl enable --now gssproxy` equivalent to:
```
# systemctl enable gssproxy
# systemctl start gssproxy
```
Because we haven't started that service after installing it (I don't see that sort of command [in the previous procedure](https://docs.theforeman.org/nightly/Installing_Server/index-satellite.html#Enrolling_Server_with_the_AD_Server_satellite)), there;s no reason to restart it, but rather simply start it.
Therefore, I'd lean towards this edit instead:
```
- # systemctl restart gssproxy # remove
  # systemctl enable --now gssproxy # keep as is
```

Reply:
https://github.com/theforeman/foreman-documentation/pull/1827#discussion_r1025343810 is the last time we discussed this. I can also confirm that I have a host that runs gssproxy even without IPA set up.

I can see this:
```console
$ systemctl show -p Wants auth-rpcgss-module
Wants=rpc-gssd.service gssproxy.service
$ systemctl show -p Wants nfs-client.target
Wants=rpc-statd-notify.service auth-rpcgss-module.service remote-fs-pre.target
```

And somehow on my system it pulls in `nfs-client.target` into the default `multi-user.target`.

So even though I don't have `/etc/krb5.keytab` it still ends up starting `gssproxy.service`.

I'd argue the restart is correct and `--now` is redundant.